src/useq/_plate.py

```diff
@@ -316,13 +316,15 @@ def selected_well_names(self) -> list[str]:
         return list(self.all_well_names[self.selected_wells].reshape(-1))
 
     def _transorm_coords(self, coords: np.ndarray) -> np.ndarray:
-        """Transform coordinates to the plate coordinate system."""
+        """Transform coordinates to the plate coordinate system and invert Y values."""
         # create homogenous coordinates
         h_coords = np.column_stack((coords, np.ones(coords.shape[0])))
         # transform
         transformed = self.affine_transform @ h_coords.T
-        # strip homogenous coordinate
-        return (transformed[:2].T).reshape(coords.shape)  # type: ignore[no-any-return]
+        transformed_coords = (transformed[:2].T).reshape(coords.shape)
+        # invert Y values
+        transformed_coords[:, 0] = -transformed_coords[:, 0]
```

Comment:
probably better to do this in the affine transform itself, just invert the scaling factor in the y axis:

```python
scaling[:2, :2] = np.diag([self.plate.well_spacing[0], -self.plate.well_spacing[1]])
```

Reply:
But is it the same? The output won't be identical...

Reply:
theoretically it should be?  did you try and it wasn't?

Reply:
yeah, it does work, but the example I gave above was xy, instead of yx...  use this:

```python
        # we invert the Y axis here to go from linearly increasing "index coordinates"
        # to cartesian "plate" coordinates (where y position decreases with increasing index)
        scale_y, scale_x = self.plate.well_spacing
        scaling[:2, :2] = np.diag([-scale_y, scale_x])
```

Reply:
yup, I tried that as well but then the test give a slightly different result and if you use a rotation and plot the plate it is rotating in the opposite direction...

Reply:
tell me what specific differences you're seeing

Reply:
Ok I think I was not convinced for another reason...I was a bit confused with the output numbers of the positions. But I think it was because we had to also consider the units of  `a1_center_xy`. I think we can say that `a1_center_xy` is in `µm` since most likely the stage coords are in µm (and the `well_points_plan` too) and we covert it during translation.

Reply:
yeah, it's confusing!  Should we consider expressing well sizes in microns too just to keep everything consistent?  Is this the only place in useq where we use mm?

Reply:
Yes I believe is the only place we use mm...you're right, maybe it is worth using µm for everything at this point...

Reply:
i know that's also problematic since all plates specs will be in mm... maybe skip that for now

Reply:
then I think this is ready for review